app/src/main/java/org/fossasia/susi/ai/skills/settings/ChatSettingsFragment.kt

```diff
@@ -247,6 +253,12 @@ class ChatSettingsFragment : PreferenceFragmentCompat(), ISettingsView {
             true
         }
 
+        manageDevices.setOnPreferenceClickListener {
+            val intent = Intent(activity, ManageDeviceActivity::class.java)
+            startActivity(intent)
```

Comment:
You are creating new activity then, a new fragment, what's the benefit then? Why don't you directly commit fragment here?

Reply:
Actually, we want to reuse this activity in different ways while setting up the smart speaker. Like configuring Spotify, sound server, location service, etc. Also, we want to come to this page fro other activities. So, it was necessary to create an activity, now this activity will be responsible for inflating all the fragments that we will have in these sections.